spring-boot-h2/spring-boot-h2-database/src/main/java/com/baeldung/h2db/demo/SpringBootApp.java

```diff
@@ -0,0 +1,60 @@
+package com.baeldung.h2db.demo;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Arrays;
+import javax.annotation.PostConstruct;
+import org.h2.tools.Server;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+
+@SpringBootApplication
+public class SpringBootApp {
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    public static void main(String[] args) {
+        SpringApplication.run(SpringBootApp.class, args);
+    }
+
+    @PostConstruct
+    private void initDb() {
+        System.out.println(String.format(
+                "****** Creating table: %s, and Inserting test data ******", "Employees"));
+
+        String sqlStatements[] = {
+                "drop table employees if exists",
```

Comment:
please correct the indentation in this section

Reply:
fixed.